### DISCUSSION TITLE
월드 심사 관련 문의

### DISCUSSION THREAD

**Discussion by hyeona-y** [naverz/zepeto-studio-kor#1145] [5. Q&A]
월드 심사 기준에 영상이 포함되면 승인 불가라고 나와있는데
현재 존재하는 월드중에 '소울 월드'에는 영상이 올라가 있더라고요
이처럼 홍보용 영상 또한도 심사 기준에 걸릴까요?
홍보용 영상을 올릴 수 있는 방법은 없을까요??

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재로서는 기업 단위의 제휴 협약을 통해서 허용을 하고 있는 점 양해 부탁 드립니다.
제휴 문의는 dl_partnerships@naverz-corp.com 을 통해서 메일 부탁 드립니다.

많은 분들이 월드 내 영상 기능을 원하고 계시기에, 검수 정책과 함께 API 제공을 논의하고 있습니다.
감사합니다.
